template/rwa-onboarding/Goerli-DssSpellCollateral.sol

```diff
@@ -61,149 +71,177 @@ contract DssSpellCollateralAction {
+        // Set "quitTo" address for RWAXXX_A_INPUT_CONDUIT_URN
+        RwaInputConduitLike(RWAXXX_A_INPUT_CONDUIT_URN).file("quitTo", RWAXXX_A_CUSTODY);
+
+        // MCD_PAUSE_PROXY and Operator permission on RWAXXX_A_INPUT_CONDUIT_JAR
+        RwaInputConduitLike(RWAXXX_A_INPUT_CONDUIT_JAR).mate(address(this));
+        RwaInputConduitLike(RWAXXX_A_INPUT_CONDUIT_JAR).mate(RWAXXX_A_OPERATOR);
+        // Set "quitTo" address for RWAXXX_A_INPUT_CONDUIT_JAR
+        RwaInputConduitLike(RWAXXX_A_INPUT_CONDUIT_JAR).file("quitTo", RWAXXX_A_CUSTODY);
+
+        // Add RWAXXX contract to the changelog
+        DssExecLib.setChangelogAddress("RWAXXX",                     RWAXXX);
+        DssExecLib.setChangelogAddress("PIP_RWAXXX",                 pip);
+        DssExecLib.setChangelogAddress("MCD_JOIN_RWAXXX_A",          MCD_JOIN_RWAXXX_A);
+        DssExecLib.setChangelogAddress("RWAXXX_A_URN",               RWAXXX_A_URN);
+        DssExecLib.setChangelogAddress("RWAXXX_A_JAR",               RWAXXX_A_JAR);
+        DssExecLib.setChangelogAddress("RWAXXX_A_INPUT_CONDUIT_URN", RWAXXX_A_INPUT_CONDUIT_URN);
+        DssExecLib.setChangelogAddress("RWAXXX_A_INPUT_CONDUIT_JAR", RWAXXX_A_INPUT_CONDUIT_JAR);
+        DssExecLib.setChangelogAddress("RWAXXX_A_OUTPUT_CONDUIT",    RWAXXX_A_OUTPUT_CONDUIT);
+
+        // Add RWAXXX to ILK REGISTRY
         REGISTRY.put(
-            "RWA999-A",
-            MCD_JOIN_RWA999_A,
-            RWA999,
-            GemJoinAbstract(MCD_JOIN_RWA999_A).dec(),
-            REG_CLASS_RWA,
+            ilk,
+            MCD_JOIN_RWAXXX_A,
+            RWAXXX,
+            decimals,
+            RWAXXX_REG_CLASS_RWA,
             pip,
             address(0),
-            // Either provide a name like:
-            "RWA999-A: Some RWA Deal",
-            // ... or use the token name:
-            // GemAbstract(RWA999).name(),
-            GemAbstract(RWA999).symbol()
+            "RWAXXX-A: TODO",
+            GemAbstract(RWAXXX).symbol()
         );
     }
 
-    function offboardCollaterals() internal {}
+    function onboardNewCollaterals() internal {
+        IlkRegistryAbstract REGISTRY     = IlkRegistryAbstract(DssExecLib.reg());
+        address MIP21_LIQUIDATION_ORACLE = DssExecLib.getChangelogAddress("MIP21_LIQUIDATION_ORACLE");
+        address MCD_PSM_USDC_A           = DssExecLib.getChangelogAddress("MCD_PSM_USDC_A");
+        address MCD_VAT                  = DssExecLib.vat();
+        address MCD_JUG                  = DssExecLib.jug();
+        address MCD_SPOT                 = DssExecLib.spotter();
+        address MCD_JOIN_DAI             = DssExecLib.daiJoin();
+
+        // --------------------------- RWA Collateral onboarding ---------------------------
+
+        // Onboard *: https://vote.makerdao.com/polling/TODO
+        onboardRwaXXX(REGISTRY, MIP21_LIQUIDATION_ORACLE, MCD_VAT, MCD_JUG, MCD_SPOT, MCD_JOIN_DAI, MCD_PSM_USDC_A);
+    }
```

Comment:
Suggestions to reflect the [latest spells cast](https://github.com/makerdao/spells-mainnet/blob/master/archive/2022-10-26-DssSpell/DssSpellCollateral.sol) (2/2):

```suggestion
```suggestion
    function collateralAction() internal {
        onboardCollaterals();
        //updateCollaterals();
        //offboardCollaterals();
    }
    
    function onboardNewCollaterals() internal {
        IlkRegistryAbstract REGISTRY     = IlkRegistryAbstract(DssExecLib.reg());
        address MIP21_LIQUIDATION_ORACLE = DssExecLib.getChangelogAddress("MIP21_LIQUIDATION_ORACLE");
        address MCD_PSM_USDC_A           = DssExecLib.getChangelogAddress("MCD_PSM_USDC_A");
        address MCD_VAT                  = DssExecLib.vat();
        address MCD_JUG                  = DssExecLib.jug();
        address MCD_SPOT                 = DssExecLib.spotter();
        address MCD_JOIN_DAI             = DssExecLib.daiJoin();

        // --------------------------- RWA Collateral onboarding ---------------------------

        // Onboard *: https://vote.makerdao.com/polling/TODO
        onboardRwaXXX(REGISTRY, MIP21_LIQUIDATION_ORACLE, MCD_VAT, MCD_JUG, MCD_SPOT, MCD_JOIN_DAI, MCD_PSM_USDC_A);
    }
    
    function updateCollaterals() internal {
        // ------------------------------- Collateral updates -------------------------------

        // Enable autoline for XXX-A
        // Poll Link:
        // Forum Link:
        // DssExecLib.setIlkAutoLineParameters(
        //    XXX-A,
        //    AMOUNT,
        //    GAP,
        //    TTL
        // );
    }

    function offboardCollaterals() internal {
        // ----------------------------- Collateral offboarding -----------------------------
        // 1st Stage of Collateral Offboarding Process
        // Poll Link:
        // uint256 line;
        // uint256 lineReduction;

        // Set XXX-A Maximum Debt Ceiling to 0
        // (,,,line,) = vat.ilks("XXX-A");
        // lineReduction += line;
        // DssExecLib.removeIlkFromAutoLine("XXX-A");
        // DssExecLib.setIlkDebtCeiling("XXX-A", 0);

        // Set XXX-A Maximum Debt Ceiling to 0
        // (,,,line,) = vat.ilks("XXX-A");
        // lineReduction += line;
        // DssExecLib.removeIlkFromAutoLine("XXX-A");
        // DssExecLib.setIlkDebtCeiling("XXX-A", 0);

        // Decrease Global Debt Ceiling by total amount of offboarded ilks
        // vat.file("Line", _sub(vat.Line(), lineReduction));

        // 2nd Stage of Collateral Offboarding Process
        // address spotter = DssExecLib.spotter();

        // Offboard XXX-A
        // Poll Link:
        // Forum Link:

        // DssExecLib.setIlkLiquidationPenalty("XXX-A", 0);
        // DssExecLib.setKeeperIncentiveFlatRate("XXX-A", 0);
        // DssExecLib.linearInterpolation({
        //     _name:      "XXX-A Offboarding",
        //     _target:    spotter,
        //     _ilk:       "XXX-A",
        //     _what:      "mat",
        //     _startTime: block.timestamp,
        //     _start:     CURRENT_XXX_A_MAT,
        //     _end:       TARGET_XXX_A_MAT,
        //     _duration:  30 days
        // });

        // Offboard XXX-A
        // Poll Link:
        // Forum Link:

        // DssExecLib.setIlkLiquidationPenalty("XXX-A", 0);
        // DssExecLib.setKeeperIncentiveFlatRate("XXX-A", 0);
        // DssExecLib.linearInterpolation({
        //     _name:      "XXX-A Offboarding",
        //     _target:    spotter,
        //     _ilk:       "XXX-A",
        //     _what:      "mat",
        //     _startTime: block.timestamp,
        //     _start:     CURRENT_XXX_A_MAT,
        //     _end:       TARGET_XXX_A_MAT,
        //     _duration:  30 days
        // });
    }
```